libraries/Microsoft.Bot.Builder.AI.QnA/Helpers/TrainHelper.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// Helper class for train API
+    /// </summary>
+    public class TrainHelper
```

Comment:
I'm not a huge fan of the "Helper" name on public classes. Consider an alternative naming scheme.